vtm/src/org/oscim/tiling/source/OkHttpEngine.java

```diff
@@ -115,4 +116,57 @@ public boolean requestCompleted(boolean success) {
 
         return success;
     }
+
+    class InputOutputStream extends InputStream {
+        InputStream mInputStream;
```

Comment:
Since we don't use internally a different `InputStream` type than the extended one, we could simplify the implementation.